acquia/cli
# Fix #657: Remove assets from phar

**Motivation**
Fixes #657

**Proposed changes**

- Remove assets from release
- Allow cache to be used when assets are missing.
- Disable fallback caching to filesystem to further prevent issues like #651

**Testing steps**

1. Follow the [contribution guide](https://github.com/acquia/cli/blob/master/CONTRIBUTING.md#building-and-testing) to build acli.phar and ensure API commands still work.

**Merge requirements**
